src/compas_fab/rhino/install.py

```diff
@@ -2,26 +2,15 @@
 from __future__ import division
 from __future__ import print_function
 
-import compas_rhino.install
+import compas.plugins
```

Comment:
What version of compas has compas.plugins?  I think you might need to update the dependency.

Reply:
good catch! I was actually a bit surprised this passes the build, but it's only because we don't test the rhino installer on CI.

The dependency needs to be updated, but right now, there's no release of `compas` containing this extension point. I was hesitating whether to add (a significant chuck of) code to handle releases before and after this extension point exists but it would only be useful for a very brief amount of time, so, I think it's best to just hold this PR until there's a release of compas, update the dependency there, and keep less code around.